Sources/XcodesKit/XcodeInstaller.swift

```diff
@@ -164,6 +164,11 @@ public final class XcodeInstaller {
             Current.shell.exit(0)
         }
     }
+
+    /// Perform the install but don't exit out but return the installed xcode version as output instead
+    public func installWithoutLogging(_ installationType: InstallationType, dataSource: DataSource, downloader: Downloader, destination: Path) -> Promise<InstalledXcode> {
```

Comment:
I keep having set back anytime I installed it

Reply:
Could you please elaborate?